### PR TITLE
acrn-config: add hybrid_rt scenario xml config for TGL

### PR DESCRIPTION
**Pull request by taoyuhong**
Add ./misc/acrn-config/xmls/config-xmls/tgl-rvp/hybrid_rt.xml

Tracked-On: #5185
Signed-off-by: Tao Yuhong <yuhong.tao@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: misc/vm_configs/xmls/config-xmls/tgl-rvp/hybrid_rt.xml [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+    <MEMORY>
+        <STACK_SIZE desc="Capacity of one stack, in bytes.">0x2000</STACK_SIZE>
+        <HV_RAM_SIZE desc="Size of the RAM region used by the hypervisor"></HV_RAM_SIZE>
+        <HV_RAM_START desc="2M-aligned Start physical address of the RAM region used by the hypervisor."></HV_RAM_START>
+        <LOW_RAM_SIZE desc="Size of the low RAM region">0x00010000</LOW_RAM_SIZE>
+        <UOS_RAM_SIZE desc="Size of the User OS (UOS) RAM.">0x200000000</UOS_RAM_SIZE>
+        <SOS_RAM_SIZE desc="Size of the Service OS (SOS) RAM.">0x400000000</SOS_RAM_SIZE>
+        <PLATFORM_RAM_SIZE desc="Size of the physical platform RAM">0x400000000</PLATFORM_RAM_SIZE>
+    </MEMORY>
```
Comment (jsun26intel) [inline misc/vm_configs/xmls/config-xmls/tgl-rvp/hybrid_rt.xml:36]: please double check SOS_RAM_SIZE/PLATFORM_RAM_SIZE, current value is not big enough to cover EPT array.

**Review by jsun26intel — CHANGES_REQUESTED** [review]

File: misc/vm_configs/xmls/config-xmls/tgl-rvp/hybrid_rt.xml (hunk 1 above)
Comment (taoyuhong) [reply misc/vm_configs/xmls/config-xmls/tgl-rvp/hybrid_rt.xml:36]: Will Fix that, change SOS_RAM_SIZE / PLATFORM_RAM_SIZ to 0x480000000

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by wenlingz** [comment]
@terryzouhao please help approve this PR, thanks

**Pull request merged**